### PR TITLE
idk part idk

### PR DESCRIPTION
**Pull request by SebastianJuncosL**

### CODE REVIEW

**Review by mateoglzc — COMMENTED** [review]
What are you doing here?

**Comment by SebastianJuncosL** [comment]
😡 algo de amplify

**Review by mateoglzc — CHANGES_REQUESTED** [review]
???

File: junqui-test-v2/amplify/backend/api/junquitestv2/cli-inputs.json [hunk 1]
```diff
@@ -7,6 +7,8 @@
     "defaultAuthType": {
       "mode": "AMAZON_COGNITO_USER_POOLS",
       "cognitoUserPoolId": "authjunquitestv2d796ad51"
-    }
+    },
+    "conflictResolution": {},
```
Comment (mateoglzc) [inline junqui-test-v2/amplify/backend/api/junquitestv2/cli-inputs.json:11]: Why add this?

**Review by mateoglzc — CHANGES_REQUESTED** [review]

File: junqui-test-v2/amplify/backend/api/junquitestv2/cli-inputs.json (hunk 1 above)
Comment (SebastianJuncosL) [reply junqui-test-v2/amplify/backend/api/junquitestv2/cli-inputs.json:11]: Just because

**Review by mateoglzc — DISMISSED** [review]

**Review by Yibizo — DISMISSED** [review]
LGTM

**Review by jorge-cab — DISMISSED** [review]
sup

**Review by jorge-cab — APPROVED** [review]
Buen cambio

**Review by mateoglzc — APPROVED** [review]

**Pull request merged**